third_party/docfx/templates/devsite/partials/uref/class.header.tmpl.partial

```diff
@@ -18,8 +18,8 @@
 {{/attributes.0}}
 {{#attributes}}
     <tr>
-      <td>{{{var_type}}}</td>
-      <td><code>{{{name}}}</code><br>{{{description}}}</td>
+      <td><span class="parametername">{{{id}}}</span></td>

```

Comment:
What is this `span` for? Do we use `class="parametername"` for styling? Seems unrelated?

If anything, I'd prefer to delete this in the ManagedReference template (in a different PR) than add it here.

Reply:
Sounds good. I'll create an issue to clean those up!